pkg/plugins/repo/service.go

```diff
@@ -75,6 +76,32 @@ func (m *Manager) GetPluginArchiveInfo(_ context.Context, pluginID, version stri
 	}, nil
 }
 
+// GetPluginJson returns the json data of the requested plugin (with optional `version`)
+func (m *Manager) GetPluginJson(_ context.Context, pluginID, version string, compatOpts CompatOpts) (*plugins.JSONData, error) {
+	v, err := m.PluginVersion(pluginID, version, compatOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	u, err := url.Parse(m.baseURL)
+	if err != nil {
+		return nil, err
+	}
+
+	u.Path = path.Join(u.Path, pluginID, "versions", v.Version)
+	data, err := m.client.SendReq(u, compatOpts)
+	if err != nil {
+		return nil, err
+	}
+	res := struct {
+		Json plugins.JSONData
+	}{}
```

Comment:
we should define a type for this.

Reply:
Should this new type also redefine `plugins.JSONData` to have a `repo.JSONData`?

Reply:
I don't think so, using `plugins.JSONData` should be fine